addons/calendar/models/calendar_attendee.py

```diff
@@ -111,6 +119,7 @@ def _send_mail_to_attendees(self, template_xmlid, force_send=False, ignore_recur
             'action_id': self.env['ir.actions.act_window'].sudo().search([('view_id', '=', calendar_view.id)], limit=1).id,
             'dbname': self._cr.dbname,
             'base_url': self.env['ir.config_parameter'].sudo().get_param('web.base.url', default='http://localhost:8069'),
+            'format_time': lambda time, time_format="medium", tz=None, lang_code=False: babel.dates.format_time(time, format=time_format, tzinfo=pytz.timezone(tz or 'UTC'), locale=lang_code),
```

Comment:
Concerning TZ support, you may add its support directly in format_time in tools so that we have a single entry point.